packages/customize-widgets/src/components/inserter/index.js

```diff
@@ -37,8 +37,7 @@ function Inserter( { setIsOpened } ) {
 					{ __( 'Add a block' ) }
 				</h2>
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					size="small"
```

Comment:
- Active an non-blocks based theme
- Visit customizer, then widgets, then click on the "+" icon button

![Screenshot 2024-10-01 at 19 09 56](https://github.com/user-attachments/assets/d03731a4-5686-4caa-b37d-4420771500be)

I picked `size="small"` to be coherent with the other "X" button (which is defined somewhere else in the codebase)